src/retro_data_structures/formats/pak_wii.py

```diff
@@ -62,4 +74,162 @@ def _emitparse_header(code: construct.CodeGen) -> str:
     _resources_end_assert=construct.Check(
         construct.this.table_of_contents.RSHD == construct.this._resources_end - construct.this._resources_start
     ),
+) 
+##TODO : Once the struct is confirmed, compile it
+
+## Not sure how this is supposed to work, is this for the whole compressed resource
+##or a single block ?
```

Comment:
as with the TODO comment above, english grammar, as opposed to french grammar, has punctuation marks directly after the character, without a space in between
```suggestion
##or a single block?
```